addons/runtime_debug_tools/scripts/debug_runtime_interaction_3d.gd

```diff
@@ -84,6 +99,10 @@ func _input(event):
         _pick_object()
 
 func _process(delta: float) -> void:
+    if _is_game_process_in_focus and _set_initial_mouse_mode:
```

Comment:
Same as in the 2d equivalent:

I think you can remove the manual focus checking you're doing here, and in `_notification`, and replace it with:

`get_window().has_focus()`